lib/perl/Genome/Qc/Tool/TestHelpers.t.d/reference.fasta

```diff
@@ -0,0 +1 @@
+/gscmnt/ams1102/info/model_data/2869585698/build106942997/Y.fa
```

Comment:
Other test data either is checked into the repository or is under `$GENOME_TEST_INPUTS` somewhere.  This one looks like the former, but really points to an MGI-specific location.  If we can't check in `Y.fa` maybe it should be under `$GENOME_TEST_INPUTS`?